### PR TITLE
refactor(api): Remove the last of apiv1 source

### LINKED ISSUE #6800: 4.0.0 Cleanup: Remove tip probe RPC handling

### LINKED ISSUE #7054: 4.0.0 cleanup: Remove the rest of APIv1
Remove the rest of the code in opentrons/api/src/opentrons/legacy_api and dependents.

### LINKED ISSUE #7064: Create PD test build deploy workflow
# Overview

We need to create a test/build/deploy github workflow (similar to the one for the runapp) so that we can move branch/tag deployments from travis to github actions.

# Implementation details

Run unit tests, e2e tests, then build and deploy to s3.

### PR DESCRIPTION
**Pull request by sfoster1**
This is the last of the core apiv1 code; more work can be done removing
handlers for it in e.g. the api implementations.

## Risk
Low, this is removing unused code

Closes #6800 
Closes #7054

### CODE REVIEW

**Comment by mcous** [comment]
Screen Shot 2020-11-30 at 10 23 24 AM 👀

File: api/src/opentrons/api/calibration.py [hunk 1]
```diff
@@ -73,43 +72,7 @@ def _set_state(self, state):
...
-                instrument=inst,
-                measured_center=measured_center)
-
-        log.info('New config: {0}'.format(config))
-
-        self._move_to_front(instrument)
+        log.warning("Deprecated call to tip_probe, update app")
         self._set_state('ready')
 
     @robot_is_busy
```
Comment (ahiuchingau) [inline api/src/opentrons/api/calibration.py:78]: We should update `pick_up_tip()` as well

**Review by ahiuchingau — CHANGES_REQUESTED** [review]
Just one comment

**Review by ahiuchingau — APPROVED** [review]
✂️

**Review by amitlissack — APPROVED** [review]
Lovely

**Pull request merged**